### PR TITLE
chore: Update sha-1 to 0.9

### PR DESCRIPTION
**Pull request by ignatenkobrain**
Signed-off-by: Igor Raits <i.gnatenko.brain@gmail.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ matrix:
     - rust: beta
     - rust: nightly
     # minimum rustc version
-    - rust: 1.39.0
+    - rust: 1.41.0
```
Comment (seanmonstar) [inline .travis.yml:12]: Can we keep the minimum rust version?

Comment (ignatenkobrain) [reply .travis.yml:12]: Not with upgrade of sha-1 :)

**Comment by conblem** [comment]
Any news on this?

**Comment by novacrazy** [comment]
Seven-month bump. @seanmonstar

**Comment by seanmonstar** [comment]
Before I was worried about increasing the minimum Rust version. But with the newest version of hyper expecting at least 1.45, this seems fine now.

**Pull request merged**